File: {{cookiecutter.project_slug}}/src/main/java/{{cookiecutter.group_id.split('.')[0]}}/{{cookiecutter.group_id.split('.')[1]}}/{{cookiecutter.mod_main_package_name}}/network/packets/ClientPacketName.java
package {{cookiecutter.group_id}}.{{cookiecutter.modid}}.network.packets;

import io.netty.buffer.ByteBuf;

public class ClientPacketName extends AbstractPacketClient{

    public String message = "";

    @Override
    public void read(ByteBuf data) throws IndexOutOfBoundsException {
        this.message = this.readString(data);
    }
}

File: src/main/java/org/dreamfinity/template/network/packets/ClientPacketName.kt
package org.dreamfinity.template.network.packets

import io.netty.buffer.ByteBuf

class ClientPacketName : AbstractPacketClient() {
    var message = ""
    override fun write(data: ByteBuf?) {
        TODO("Not yet implemented")
    }

    @Throws(IndexOutOfBoundsException::class)
    override fun read(data: ByteBuf?) {
        message = readString(data!!)
    }
}